src/targets/upm.ts

```diff
@@ -0,0 +1,141 @@
+import * as Github from '@octokit/rest';
+import simpleGit from 'simple-git';
+import {
+  getAuthUsername,
+  getGithubApiToken,
+  getGithubClient,
+  GithubRemote,
+} from '../utils/githubApi';
+
+import { logger as loggerRaw } from '../logger';
+import { TargetConfig } from '../schemas/project_config';
+import { BaseTarget } from './base';
+import {
+  BaseArtifactProvider,
+  RemoteArtifact,
+} from '../artifact_providers/base';
+import { reportError } from 'src/utils/errors';
+import { extractZipArchive } from 'src/utils/system';
+import { withTempDir } from '../utils/files';
+import { isDryRun } from 'src/utils/helpers';
+
+const logger = loggerRaw.withScope('[upm]');
+
+const DEFAULT_RELEASE_REMOTE: GithubRemote = new GithubRemote("getsentry", "unity");
```

Comment:
Best not to default like this, so a misconfigured release ends up overriding `getsentry/unity`. Let's just fail if the right configuration isn't provided.